src/main/resources/com/suse/matcher/rules/optaplanner/Scores.drl

```diff
@@ -23,20 +24,26 @@ global HardSoftScoreHolder scoreHolder;
 
 rule "calculate12Penalties"
   when
-    PenaltyGroup($penaltyGroupId: id)
-    Subscription($subscriptionId : id, policy != null && policy == Policy.ONE_TWO, ignored == false)
-    accumulate (
-        // count the number of VMs that use a certain 1:2 subscription in one penalty group
-        PenaltyGroup($guestId : guestId, id == $penaltyGroupId) and
-        PartialMatch($groupId : groupId, systemId == $guestId, subscriptionId == $subscriptionId) and
-        Match(id == $groupId, confirmed == true);
-        $count : count();
-        // if it is odd...
-        ($count % 2) == 1
-    )
+    Subscription($subscriptionId : id, policy != null && policy == Policy.ONE_TWO, ignored == false) and
+    PenaltyGroup($penaltyGroupId: id, $guestId : guestId) and
+    PartialMatch($groupId : groupId, systemId == $guestId, subscriptionId == $subscriptionId) and
+    Match(id == $groupId, confirmed == true);
   then
     // ...add a 50 cent penalty. This ensures a whole number of subscriptions is
     // always used for the same penalty group
+    insertLogical(new ConfirmedMatchInPenaltyGroup($subscriptionId, $penaltyGroupId, $guestId));
```

Comment:
It seems to me that the code does not agree with the comment any more, please reword the latter to better explain what you are doing, thanks!